packages/manager/components/chart/aggregate-chart.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  ChartDonut, Chart,
+  ChartAxis,
+  ChartGroup,
+  ChartLine,
+  ChartVoronoiContainer,
+} from "@patternfly/react-charts";
+import React, { FunctionComponent } from "react";
+import { AnyProps } from "../models/props";
+
+const AggregateChart: FunctionComponent<AnyProps> = ({ type, props }: AnyProps) => {
+  if (type === "donut")
```

Comment:
if (type === "donut") It seems the check value is hard coded. Is it normal? @soumyadip007 @hybridx

Reply:
We can create an object here

Reply:
chartType.donut and chartType.month

Reply:
@arkaprovob  please check now,